wgpu-hal/src/gles/egl.rs

```diff
@@ -800,24 +800,17 @@ impl crate::Instance<super::Api> for Instance {
                 }
             }
             #[cfg(not(feature = "emscripten"))]
-            Rwh::Wayland(_) => {
+            (Rwh::Wayland(_), raw_window_handle::RawDisplayHandle::Wayland(display_handle)) => {
```

Comment:
Let's add a second using for this as well, so we can do Rdh::Wayland(display_handle) and keep it all on one line.

So the using at the top of the function would be:

```rust
use raw_window_handle::{RawWindowHandle as Rwh, RawDisplayHandle as Rdh};
```